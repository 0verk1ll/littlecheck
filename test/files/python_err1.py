# RUN: /usr/bin/python %s

import sys

sys.stderr.write("ALPHA\n")
# CHECKERR: BETA

