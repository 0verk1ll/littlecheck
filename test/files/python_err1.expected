Failure in python_err1.py:

  The CHECKERR on line 6 wants:
    BETA

  which failed to match line stderr:1:
    ALPHA

  when running command:
    /usr/bin/python python_err1.py
